app/controllers/application_controller.rb

```diff
@@ -96,64 +96,53 @@ def success_message(obj_name, action)
     "#{_('Successfully %{action} your %{object}.') % {object: obj_name, action: action}}"
   end
 
-  # Check whether the string is a valid array of JSON objects
-  def is_json_array_of_objects?(string)
```

Comment:
As far as I can see, `is_json_array_of_objects?` is not used within the code. Removed.